seatunnel-server/seatunnel-scheduler/seatunnel-scheduler-dolphinscheduler/src/main/java/org/apache/seatunnel/scheduler/dolphinscheduler/dto/ProcessDefinitionDto.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.scheduler.dolphinscheduler.dto;
+
+import org.apache.seatunnel.server.common.DateUtils;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.Date;
+import java.util.Map;
+
+@Data
+@Builder
+public class ProcessDefinitionDto {
+    private int id;
+    private long code;
+    private String name;
+    private String releaseState;
+    private long projectCode;
+    private String description;
+    private Date createTime;
+    private Date updateTime;
+    private String userName;
+    private String projectName;
+    private String locations;
+    private String scheduleReleaseState;
+    private int timeout;
+    private int tenantId;
+    private String tenantCode;
+    private String modifyBy;
+    private int warningGroupId;
+
+    public static ProcessDefinitionDto fromMap(Map<String, Object> map) {
```

Comment:
We already defined a ProcessDefinitionDto here, why still need to use `Map<String, Object>`?

Reply:
This map is get from response data and then transform to java pojo. 
And why we use map to send request or receive? Due to if we use pojo to receive data [use Json to deserialization response data], when the data structure changed , it may throws exceptions. it's the same reason why i used map while send request.

Reply:
OK, i will change Map to Java pojo